keepassxc-browser/options/options.js

```diff
@@ -562,9 +562,14 @@ options.initSitePreferences = function() {
 
 options.initAbout = function() {
     const version = browser.runtime.getManifest().version;
+    let platform = navigator.platform;
+    if (platform === 'Win32' && navigator.userAgent.includes('x64') || navigator.userAgent.includes('WOW64')) {
```

Comment:
Recommend wrapping the or'd conditions:

```suggestion
    if (platform === 'Win32' && (navigator.userAgent.includes('x64') || navigator.userAgent.includes('WOW64'))) {
```